Makefile.include

```diff
@@ -575,3 +564,4 @@ CFLAGS += -include '$(RIOTBUILD_CONFIG_HEADER_C)'
 
 # include multislot support
 include $(RIOTMAKE)/multislot.mk
+endif # BOARD=none
```

Comment:
The indentation above seems off (not related to this PR though)

Reply:
Actually, it is. But I didn't want to indent the whole file (or most of it), so I intentionally didn't indent the "BOARD!=none" case.

Reply:
ok